src/app/pages/property-details/reviews/reviews.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Subscription } from 'rxjs';
+import { Component, OnInit, Input } from '@angular/core';
+import { ClientReviewsService } from 'src/app/services/client-reviews/client-reviews.service';
+import { Router } from '@angular/router';
+import { NgxSpinnerService } from 'ngx-spinner';
+import { APPCONFIG } from 'src/app/config';
+
+
+
+@Component({
+  selector: 'app-reviews',
+  templateUrl: './reviews.component.html',
+  styleUrls: ['./reviews.component.scss']
+})
+export class ReviewsComponent implements OnInit {
+  @Input() clientId: number;
+  subscription: Subscription = new Subscription();
+  id: number;
```

Comment:
Great work once again @dorothyas , I seem to not see where you are using id, review, reviewer, replies and image.

Reply:
I have also corrected this. Thanks